boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES

```diff
@@ -29,8 +29,9 @@ aliases:
     - devppratik
   srep-functional-team-orange:
     - bng0y
+    - bergmannf
     - typeid
-    - Makdaam
```

Comment:
Not sure about this: @Makdaam can you confirm?

Reply:
I thought @Makdaam's off to a new team as well :D